src/Sarif/Autogenerated/Address.cs

```diff
@@ -6,7 +6,9 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Runtime.Serialization;
+
```

Comment:
Should we add the new grouping for autogenerated files?

Reply:
already fixed

---
In reply to: [510432670](https://github.com/microsoft/sarif-sdk/pull/2125#discussion_r510432670) [](ancestors = 510432670)